src/pages/MainPage/index.js

```diff
@@ -0,0 +1,37 @@
+import './style.scss';
+import {Link, useNavigate} from "react-router-dom";
+import Button from "../../atoms/Button";
+import React, {useState} from "react";
+import FeedbackModal from "../../molecules/FeedbackModal";
+import { observer } from 'mobx-react-lite';
+import userStore from "../../store";
+
+function MainPage() {
+    const [modalActive, setModalActive] = useState(false);
+    const navigate = useNavigate();
+    const jwtToken = userStore.jwt;
+
+    if (jwtToken) {
+        const tokenData = JSON.parse(atob(jwtToken.split('.')[1]));
+        const currentTime = Date.now() / 1000;
+
+        if (tokenData.exp && tokenData.exp < currentTime) {
+            navigate('/home');
```

Comment:
in this scenario we should also delete expired token
```suggestion
            userStore.logout()
```